### DISCUSSION TITLE
CSS files

### DISCUSSION THREAD

**Discussion by Serenoiia** [Matthew-Tangk/project-tech-A2#40] [Ideas]
Ik zat te denken. We wouden allemaal onze feature een aparte CSS file maken. Maar dan staan er in de head 5 css files. Is het anders een idee om te kijken of we sass kunnen gebruiken. Daarmee kan je volgens mij allemaal sass bestanden toevoegen aan 1 css bestand.

**Comment by Juul03** [comment]
Klinkt als een goed plan! Moeten we inderdaad een bundler gebruiken om de performance te verbeteren!
